packages/twenty-front/vite.config.ts

```diff
@@ -18,6 +19,7 @@ export default defineConfig(({ command, mode }) => {
     VITE_BUILD_SOURCEMAP,
     VITE_DISABLE_TYPESCRIPT_CHECKER,
     VITE_DISABLE_ESLINT_CHECKER,
+    REACT_APP_BASE_URL,
```

Comment:
Why do we introduce this url? Seems like we don't use it in the end?
I think it might add confusion to people who think they have to feel it while it's not a requirement? 
Should we do something more SSL+Vite specific instead like VITE_ENABLE_SSL?

Reply:
You're right I will change that

Reply:
Done